web/migrations/0059_alter_educationalvideo_description.py

```
@@ -0,0 +1,18 @@
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("web", "0058_forumtopic_github_issue_url_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="educationalvideo",
+            name="description",
+            field=models.TextField(blank=True, help_text="Optional – describe what viewers will learn from this video"),
```

Ruff (0.8.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)